Makefile

```diff
@@ -1,3 +1,10 @@
+ifeq ($(OS),Windows_NT)
+SEPARATOR := ;
+else
+SEPARATOR := :
+endif
+
+
```

Comment:
Minor but the SEPARATOR variable might be confused with path separator `/` or `\` especially with how it's used below - may I suggest ENV_PATH_SEP or something similar.